install/installer/cmd/testdata/render/pat-key/config.yaml

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+apiVersion: v1
+domain: gitpod.example.com
```

Comment:
What is the purpose of this test? It's the same as the `minimal` test, right?

Is this …

1. a left-over, or
2. a test where we should add a custom value for the `personalAccessTokenSigningKey` config?